ghost/core/core/server/data/schema/schema.js

```diff
@@ -538,7 +538,10 @@ module.exports = {
                 isIn: [['member', 'import', 'system', 'api', 'admin']]
             }
         },
-        batch_id: {type: 'string', maxlength: 24, nullable: true}
+        batch_id: {type: 'string', maxlength: 24, nullable: true},
+        '@@INDEXES@@': [
```

Comment:
I think we should just add `index: true` to `attribution_id`. `@@INDEXES@@` also works but it's primarily for composite indexes

Reply:
Oh, I completely missed that this was possible. I'll update it

Reply:
Updated ✅